CreeDictionary/CreeDictionary/migrations/0003_add_wordform_paradigms.py

```diff
@@ -0,0 +1,64 @@
+from django.db import migrations
+
+WORDFORM_TO_PARADIGM = {
+    # Personal pronouns
+    ("niya", "PRON"): "personal-pronouns",
+    ("kiya", "PRON"): "personal-pronouns",
+    ("wiya", "PRON"): "personal-pronouns",
+    ("niyanân", "PRON"): "personal-pronouns",
+    ("kiyânaw", "PRON"): "personal-pronouns",
+    ("kiyawâw", "PRON"): "personal-pronouns",
+    ("wiyawâw", "PRON"): "personal-pronouns",
+    # Animate demonstratives
+    ("awa", "PRON"): "demonstrative-pronouns",
+    ("ana", "PRON"): "demonstrative-pronouns",
+    ("nâha", "PRON"): "demonstrative-pronouns",
+    ("ôki", "PRON"): "demonstrative-pronouns",
+    ("aniki", "PRON"): "demonstrative-pronouns",
+    ("nêki", "PRON"): "demonstrative-pronouns",
+    # Inanimate demonstratives
+    ("ôma", "PRON"): "demonstrative-pronouns",
+    ("ôhi", "PRON"): "demonstrative-pronouns",
+    ("anima", "PRON"): "demonstrative-pronouns",
+    ("anihi", "PRON"): "demonstrative-pronouns",
+    ("nêma", "PRON"): "demonstrative-pronouns",
+    ("nêhi", "PRON"): "demonstrative-pronouns",
+    # Inanimate/Obviative inanimate demonstratives
+    ("ôhi", "PRON"): "demonstrative-pronouns",
+    ("anihi", "PRON"): "demonstrative-pronouns",
+    ("nêhi", "PRON"): "demonstrative-pronouns",
```

Comment:
@UAlbertaALTLab/crk-fst I would appreciate a review to make sure I didn't missing any wordforms that should belong to the given paradigms. These are the basic demonstrative pronouns.

C.f., https://plainscree.atlas-ling.ca/grammar/words/pronouns/demonstrative-pronouns/#:~:text=Inanimate-,demonstrative,-Pronouns

Reply:
do you want emphatic/contrastive pronouns like "nista" etc?

Reply:
I think we'll have a different paradigm for nîsta/kîsta/wîsta. Right now, we only have paradigms for "basic" demonstratives.

Reply:
And we'd want to check that the FST analyzes those properly as well.

Reply:
The FST gets _funky_ with pronouns. There's a lot of inconsistency and duplicate results. Do the mappings above **look correct** such that I can merge this and get the first draft static paradigms working?